packages/application-shell/src/components/application-shell-authenticated/application-shell-authenticated.tsx

```diff
@@ -229,12 +229,8 @@ export const ApplicationShellAuthenticated = (
                               css={css`
                                 height: 100vh;
                                 display: grid;
-                                grid-template-rows: ${isNewNavigationEnabled
-                                  ? `auto ${DIMENSIONS.header} 1fr`
-                                  : `auto ${DIMENSIONS.header} 1fr`};
-                                grid-template-columns: ${isNewNavigationEnabled
-                                  ? 'minmax(80px, 256px) 1fr'
-                                  : 'auto 1fr'};
+                                grid-template-rows: auto ${DIMENSIONS.header} 1fr;
+                                grid-template-columns: auto 1fr;
```

Comment:
I reverted those changes as using `minmax()` here was causing buggy behaviour (in my opinion) which can be observed in the screencast in the description👆(beige background was visible when changing size of the viewport in between 80px and 256px wide side panel). 

But it's probable are I don't have the full understanding why it was introduced.

Reply:
Maybe @chloe0592 can share something here.

However I don't really get what's the problem. Is it that the change from collapsed to expanded should happen before?
The background color should not be a problem because it will be white. I guess is now beige only for debugging purposes.

Reply:
> However I don't really get what's the problem.

My point is that right now we are using 2 widths for the navbar (either 80px or 256px) and the side panel's width is `auto`. And I think that works fine. 

Using minmax() makes the side panel flexible in that range, but navbar doesn't fill up that space. So the white space seems to be redundant.

Reply:
@kark feel free to iterate these stylings as you need. The changes were indeed made only for the purpose of having a 'visible' container where the new navigation will be placed.